Contribution guideline link

jackcarlisle opened the pull request:
* adds CONTRIBUTING file in .github as link to contribution repo for future examples 

#8

iteles commented:
<3 this.

nelsonic commented:
@jackcarlisle I _think_ it needs to be `CONTRIBUTING.md` (_markdown_)
(_or at least that's what most of the **examples** are using_: https://github.com/blog/1184-contributing-guidelines )

jackcarlisle commented:
@nelsonic I've changed it to `CONTRIBUTING.md` and moved it into the [other PR](https://github.com/dwyl/contributing/pull/10). This one can now be closed
